packages/api/src/cache/clients/RedisClient.ts

```diff
@@ -1,5 +1,4 @@
-import type { RedisClientType } from '@redis/client'
-import type { RedisClientOptions } from 'redis'
+import type { RedisClientType, RedisClientOptions } from 'redis'
```

Comment:
`redis` installs and exports all of `@redis/client`: https://www.npmjs.com/package/redis?activeTab=explore